### ISSUE TITLE
Error decrypting: the Mac is invalid

### ISSUE THREAD

**Issue by snehamanukonda** [grokability/snipe-it#8001]
Hi,
I am getting error as invalid mac while displaying the encrypted assets values. App_key is not changed. Can you please help me out in this.

**Comment by snehamanukonda** [comment]
Once I applied php artisan key: generate command. Now I copied the App_key from previous .env file to the current .env file and I applied the following commands. Still I am getting error Decrypting: the Mac is invalid. Any suggestions !!

composer dump-autoload
composer clear-cache
php artisan config:clear
php artisan config:clear
php artisan config:cache

**Comment by figureone** [comment]
This one just hit us too, after an upgrade from 4.7.7 to 4.9.4. Still investigating, but can provide details if needed.

**Comment by figureone** [comment]
Just following up, after reviewing Snipe-IT backups in `storage/app/backups`, I found that the `APP_KEY` in `.env` had been changed about 6 months ago, but I believe the config cache was never cleared so Snipe-IT continued to use the old value. When we upgraded to 4.9.4 last week, the config cache was likely cleared, so the new `APP_KEY` started getting used, causing all decryptions to fail. Reverting to the old `APP_KEY` seems to have solved the problem.
